src/workerd/jsg/fast-method-test.c++

```diff
@@ -0,0 +1,304 @@
+#include "jsg-test.h"
+
+namespace workerd::jsg::test {
+namespace {
+
+class FastMethodContext: public jsg::Object, public jsg::ContextGlobal {
+ public:
+  int32_t add(int32_t a, int32_t b) {
+    return a + b;
+  }
+
+  int32_t processValue(v8::Local<v8::Value> value) {
+    // Return 42 if it's a number, 0 otherwise
+    if (value->IsNumber()) {
+      return 42;
+    }
+    return 0;
+  }
+
+  // Keep this test as it is, and don't change it to use kj internals to test this
```

Comment:
What does this comment mean?

Reply:
TLDR: When type wrapper is supported, don't change line 22 to use kj::String because we're testing Local<Object>

Reply:
Ah, maybe add "When arbitrary type unwrapping is supported" to the comment.